src/node.c

```diff
@@ -377,6 +378,7 @@ const char *cmark_node_get_literal(cmark_node *node) {
   case CMARK_NODE_HTML_INLINE:
   case CMARK_NODE_CODE:
   case CMARK_NODE_FOOTNOTE_REFERENCE:
+  case:
```

Comment:
These changes in `node.c` look weird and I don't think they're needed. Please could you revert them?

Reply:
```suggestion
  case CMARK_NODE_FOOTNOTE_DEFINITION:
```